src/qt/optionsmodel.cpp

```diff
@@ -83,6 +83,11 @@ void OptionsModel::Init(bool resetSettings)
         settings.setValue("fCoinControlFeatures", false);
     fCoinControlFeatures = settings.value("fCoinControlFeatures", false).toBool();
 
+    if (!settings.contains("m_enable_psbt_controls")) {
+        settings.setValue("m_enable_psbt_controls", false);
+    }
+    m_enable_psbt_controls = settings.value("m_enable_msbt_controls", false).toBool();
```

Comment:
In commit "qt: hide Create Unsigned button behind an expert mode option" (9d8cf228a339101cc4956b32b39e377f5b5c21be)

Bug: spelling "msbt" should be "psbt".

Separate, minor suggestion: I'd also suggest dropping the "m_" prefix in the setting name (not member name). Other setting names don't have "m_" prefixes. Settings names are persisted externally and survive across releases, while member names are changeable implementation details. Also (this part is just my opinion), having redundant `OptionsModel` member variables like `m_enable_psbt_controls` and `fCoinControlFeatures` even exist is a mistake and a footgun to begin with. `OptionsModel` only needs member variables for specific settings  with complicated external interactions, like the pruning setting, not for plain settings that can be get and set with `data()` and `setData()` calls. I'm not suggesting getting rid of the members in this PR, just saying it would be strange later if the members were simplified away and some settings still had m_ prefixes.